Lesson23/Mesut/filter.js

```diff
@@ -353,8 +376,13 @@ const studentsExercise9 = [
 	},
 ];
 
+const studentsExercise9Result=studentsExercise9.filter(student=>student.grade=="A"||student.grade=="A-"||student.grade=="B+"||student.grade=="B"||student.grade=="B-"||student.grade=="C")
```

Comment:
This code would work for most of the cases but not all.
Grades could be like this: "A", "A+", "A++", "A-", "A--" so it would be better to account for all the cases. If you see, all the A grades start with the actual grade and then (a combination of) + or - following.
One approach of solving it could be to do like this:
studentsExercise9Result=studentsExercise9.filter(student=>student.grade[0].toLowerCase() === "a" || student.grade[0].toLowerCase() === "b" || student.grade[0].toLowerCase() === "c");
Here I get the first character of the grade string and convert it to the lower case to avoid error based on the case.
Then, I use a strict comparison operator to compare it to one of the main passing grades.